src/_macosx.m

```diff
@@ -1902,6 +1902,7 @@ -(void)save_figure:(id)sender
       case 1: [[NSCursor arrowCursor] set]; break;
       case 2: [[NSCursor crosshairCursor] set]; break;
       case 3: [[NSCursor openHandCursor] set]; break;
+      case 4: break;
```

Comment:
Can we get a comment here about the fact that OS X handles the busy for us and doesn't provide an icon?